src/org/aavso/tools/vstar/util/prefs/NumericPrecisionPrefs.java

```diff
@@ -156,10 +187,95 @@ public static DecimalFormat getOtherOutputFormat() {
 		return getOutputFormat(otherDecimalPlaces, Type.OTHER);
 	}
 
+	public static String formatOtherLocaleIndependent(double num) {
+		return getOtherOutputFormatLocaleIndependent().format(num);
+	}
+	
+	public static DecimalFormat getOtherOutputFormatLocaleIndependent() {
+		return getOutputFormatLocaleIndependent(otherDecimalPlaces, Type.OTHER);
+	}
+	
 	public static String getOtherInputFormat() {
 		return getInputFormatString(otherDecimalPlaces, Type.OTHER);
 	}
 
+	// PMAK (2021-03-25): 
+	// Special case: format for polynomial coefficients.
+	// These coefficients have a huge value range and cannot be satisfactorily
+	// represented by DecimalFormat.
+	// What format would be better: general (%.xxG) or pure scientific (%.xxE)?
+	// Both have issues (for the current VeLa numeric parser):
+	//   1) signed positive exponent cannot be correctly parsed by VeLa now (i.e. 1.234E+15)
+	//   2) long number strings without decimal separator cannot be parsed too (i.e. -608516245008941)
```

Comment:
@mpyat2 Thanks. I'm adding NumberParser unit tests to look at this. I'll either address it in #138 or in a separate issue. VeLa integers are currently Java ints. Should be longs. For now, the workaround would be `-608516245008941.0` to force VeLa to parse it as a real.

Reply:
ok